### PR TITLE
[ANDROAPP-5165] Display accuracy indicator

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: dhis2_android_maps/src/main/java/org/dhis2/maps/managers/DefaultMapManager.kt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            )
+        }
+    }
+
+    override fun findFeature(
+        source: String,
+        propertyName: String,
+        propertyValue: String,
+    ): Feature? {
+        return null
```
Comment (andresmr) [inline dhis2_android_maps/src/main/java/org/dhis2/maps/managers/DefaultMapManager.kt:109]: is this the expected return value?

File: app/src/main/java/org/dhis2/App.java [hunk 2]
```diff
@@ -128,7 +124,7 @@ public void onCreate() {
     }
 
     public void initCrashController() {
-        if (areTrackingPermissionGranted()) {
+        if (false/*areTrackingPermissionGranted()*/) {
```
Comment (mmmateos) [inline app/src/main/java/org/dhis2/App.java:127]: Why this is commented?

File: dhis2_android_maps/src/main/java/org/dhis2/maps/views/MapSelectorActivity.kt [hunk 3]
```diff
@@ -1,96 +1,129 @@
...
 
     private var initialCoordinates: String? = null
-
-    private val baseMapManager by lazy {
-        BaseMapManager(this, emptyList())
+    private lateinit var mapManager: DefaultMapManager
+
+    @Suppress("UNCHECKED_CAST")
+    private val mapSelectorViewModel: MapSelectorViewModel by viewModels<MapSelectorViewModel> {
+        object : ViewModelProvider.Factory {
```
Comment (mmmateos) [inline dhis2_android_maps/src/main/java/org/dhis2/maps/views/MapSelectorActivity.kt:101]: Not possible to inject this ViewmodelFactory?

File: dhis2_android_maps/src/main/java/org/dhis2/maps/managers/DefaultMapManager.kt (hunk 1 above)
Comment (Balcan) [reply dhis2_android_maps/src/main/java/org/dhis2/maps/managers/DefaultMapManager.kt:109]: Yes. The DefaultMapManager use to campture coordinates has no "find feature" support but it needs to implement those methods.

**Review by mmmateos — APPROVED** [review]

**Pull request merged**